src/gpgpu-sim/gpu-cache.h

```diff
@@ -563,10 +563,12 @@ class cache_config {
     char ct, rp, wp, ap, mshr_type, wap, sif;
 
     int ntok =
-        sscanf(config, "%c:%u:%u:%u,%c:%c:%c:%c:%c,%c:%u:%u,%u:%u,%u", &ct,
```

Comment:
Just reformatting the newlines, so that each cluster of params surrounded by commas are in one line